apps/SparseMatrix/matrix_operation.cuh

```diff
@@ -0,0 +1,379 @@
+#pragma once
```

Comment:
Please document these functions at least explain what they do and please try to follow the style in other files like `rxmesh.h`